app/views/users/login.html.erb

```diff
@@ -6,6 +6,17 @@
         <%= image_tag("login_logo.png") %>
       </div><!--/#brand-->
     </div>
+    <% if flash_inline.inspect %>
```

Comment:
wouldn't this always be true?

Reply:
Should be perhapse `flash_inline.present?`

Reply:
Changed it to `.any?` overlooked that as it wouldn't show any notification anyways when looping.